oe-account-service/op-energy-account-service/src/OpEnergy/BlockTimeStrike/Server/V1/BlockTimeStrikeGuessService.hs

```diff
@@ -260,27 +260,55 @@ getBlockTimeStrikesGuessesPage mpage mfilter = profile "getBlockTimeStrikesGuess
   latestUnconfirmedBlockHeightV <- asks (BlockTime.latestUnconfirmedBlockHeight . blockTimeState)
   mlatestUnconfirmedBlockHeight <- liftIO $ TVar.readTVarIO latestUnconfirmedBlockHeightV
   configBlockTimeStrikeGuessMinimumBlockAheadCurrentTip <- asks (configBlockTimeStrikeGuessMinimumBlockAheadCurrentTip . config)
+  latestConfirmedBlockV <- asks (BlockTime.latestConfirmedBlock . blockTimeState)
+  mlatestConfirmedBlock <- liftIO $ TVar.readTVarIO latestConfirmedBlockV
   recordsPerReply <- asks (configRecordsPerReply . config)
   let
       linesPerPage = maybe recordsPerReply (maybe recordsPerReply id . blockTimeStrikeGuessResultPublicFilterLinesPerPage . fst . unFilterRequest ) mfilter
-  case mlatestUnconfirmedBlockHeight of
-    Nothing -> do
-      let msg = "getBlockTimeStrikesGuessesPage: no confirmed block found yet"
-      runLogging $  $(logError) msg
-      throwJSON err500 msg
-    Just latestUnconfirmedBlockHeight -> do
+  case (mlatestUnconfirmedBlockHeight, mlatestConfirmedBlock) of
```

Comment:
What about the other 3 cases? If they are errors, can you describe? Sorry if I missed them.